### DISCUSSION TITLE
有办法给重定向到域名`/admin`下面吗？

### DISCUSSION THREAD

**Discussion by JIeJaitt** [Qexo/Qexo#530] [Q&A] [answered]
如题，但是我的这个域名绑定了vercel其他的项目，能不能通过修改代码来实现呢？
不太懂前端，希望大佬指路

**Comment by bobevans-commits** [comment]
建一个二级域名呗

**Reply by JIeJaitt** [reply]
啥意思，大佬，建一个二级域名跟这个有啥关系吗？

**Comment by am-abudu** [comment, marked as answer]
目前没有办法

**Reply by JIeJaitt** [reply]
不能通过自定义`vercel.json`里面的路由参数来决定吗？

**Reply by am-abudu** [reply]
可以，但是你这样根目录并没有办法放东西，所以没什么意义

**Reply by JIeJaitt** [reply]
我的博客也是部署在vercel的，我想让我的博客`域名/admin`可以访问在线编辑器
